RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/decorator/StructurePathDecorator.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Created by Grzegorz Małopolski on 10/6/21, 11:28 AM
+ * Copyright © 2021 Ringier Axel Springer Tech. All rights reserved.
+ *
+ */
+
+package com.ringpublishing.tracking.internal.decorator
+
+import com.ringpublishing.tracking.data.Event
+import com.ringpublishing.tracking.internal.constants.Constants
+import com.ringpublishing.tracking.internal.delegate.ConfigurationManager
+
+internal class StructurePathDecorator(private val configurationManager: ConfigurationManager) : BaseDecorator()
+{
+	private val applicationRootPath get() = configurationManager.ringPublishingTrackingConfiguration.applicationRootPath
```

Comment:
a czemu to nie moze byc inline, raz to jest uzyte

Reply:
Dla bezpieczeństwa.
W razie jeśli obiekt byłby stworzony szybciej, niż inicjalizacja konfiguracji aplikacji.